packages/gatsby/src/bootstrap/load-themes/index.js

```diff
@@ -8,6 +9,8 @@ import { getConfigFile } from "../get-config-file"
 const { resolvePlugin } = require(`../load-plugins/load`)
 const reporter = require(`gatsby-cli/lib/reporter`)
 
+const createScopedRequire = createRequire || createRequireFromPath
```

Comment:
We can use `const { createRequireFromPath } = require(`gatsby-core-utils`)` - we already use it in parts of code (specifically in `load-plugins` which you adjust as well in this PR)

Reply:
Just pushed the update.

I did see that it was used in `load-plugins`, but I overlooked that it was using `gatsby-core-utils`.  Just thought it was using `module`.